docker-compose.yml

```diff
@@ -184,6 +184,7 @@ services:
 
     httpbin_local:
       image: kennethreitz/httpbin@sha256:2c7abc4803080c22928265744410173b6fea3b898872c01c5fd0f0f9df4a59fb
+      platform: linux/amd64
```

Comment:
I had to add this line in order to run `docker-compose up -d httpbin_local`, it is in a number of entries in this file above as well. (Happy to take it out if we want to keep this PR scoped)